### PR TITLE
feature: Convert Artist(s) Routes

### PR DESCRIPTION
**Pull request by icirellik**
This change converts the `/artist/*` and `/artists` routes to the NOVO template.

### CODE REVIEW

**Review by joeyAghion — APPROVED** [review]
Looks good to me other than the related test failure. 🙏

File: src/lib/middleware/sharifyLocals.ts [hunk 1]
```diff
@@ -47,5 +47,7 @@ export function sharifyLocalsMiddleware(
 
   res.locals.sd.IS_GOOGLEBOT = Boolean(ua.match(/Googlebot/i))
 
+  res.locals.sd.REFERRER = req.get("Referrer")
```
Comment (damassi) [inline src/lib/middleware/sharifyLocals.ts:50]: Loving this `sharifyLocals` file 👍

**Review by damassi — APPROVED** [review]
A+

**Comment by icirellik** [comment]
The artist signup modal adds about 20KB to the bundle size.

**Pull request merged**